tuxemon/states/persistance/save_menu.py

```diff
@@ -32,15 +33,37 @@ def initialize_items(self):
         slot_rect = Rect(0, 0, rect.width * 0.80, rect.height // 6)
         for i in range(self.number_of_slots):
             # Check to see if a save exists for the current slot
-            if os.path.exists(prepare.SAVE_PATH + str(i + 1) + ".save"):
-                image = self.render_slot(slot_rect, i + 1)
-                item = MenuItem(image, T.translate("menu_save"), None, None)
-                self.add(item)
+            if cfgcheck.compress_save:
+                if cfgcheck.csave_lzma:
+                    if os.path.exists(prepare.SAVE_PATH + str(i + 1) + ".csave.lza"):
```

Comment:
the only change here is the filename, it appears.  i don't see a need for duplicating everything

Reply:
best way I know how to do this if you know a better way I am all ears

Reply:
It would be something like:
```python
# save_format should be a string containing the name of the compression module or ``None`` for no compression

# I propose to use as the extension the compression module name, so that no mapping "compression" <-> "extension" is required

save_suffix = "save" if save_format is None else f"csave.{save_format}"

save_name = f"{prepare.SAVE_PATH}{i + 1}.{save_suffix}"

if os.path.exists(save_name):
    image = self.render_slot(slot_rect, i + 1)
    item = MenuItem(image, T.translate("menu_save"), None, None)
    self.add(item)
else:
    if not empty_image:
        empty_image = self.render_empty_slot(slot_rect)
    item = MenuItem(empty_image, "SAVE", None, None)
    self.add(item)
```